lib/register.coffee

```diff
@@ -15,37 +15,38 @@ limitations under the License.
 ###
 
 Promise = require('bluebird')
+postAsync = Promise.promisify(require('request').post, multiArgs: true)
 _ = require('lodash')
 
+randomHexChar = ->
+	return _.random(15).toString(16)
```

Comment:
[Something like his](https://github.com/resin-io/resin-supervisor/blob/master/src/lib/random-hex-string.coffee) should work instead.